src/wellKnownDidConfiguration/index.ts

```diff
@@ -70,16 +76,16 @@ export async function createCredential(
 
   const credential = Credential.fromClaim(claim)
 
-  const assertionKey = document.assertionMethod?.[0]
-
-  if (!assertionKey) {
-    throw new Error('Full DID doesnt have assertion key: Please add assertion key')
-  }
-
-  return Credential.createPresentation({
+  const presentation = await Credential.createPresentation({
     credential,
     signCallback,
   })
+
+  if (presentation.claimerSignature.keyUri !== document.uri + assertionKey.id) {
```

Comment:
I believe the preferred way to concatenate strings is to use template strings, is this right?
```suggestion
  if (presentation.claimerSignature.keyUri !== `${document.uri}${assertionKey.id}`) {
```

Reply:
god I'm so used to eslint fixing this that I just write it like this now :D seems to be an extra rule though bc here it's ignored